### PR TITLE
added background music

### PR DESCRIPTION
**Pull request by vaibhavviking**
there is background music with play/pause button. music is located in assets/sound.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -230,6 +231,26 @@ function getHighScore() {
   }
 }
 
+function startbgm(){
+  var x = document.getElementById("player");
```
Comment (indremak) [inline index.js]: Please use `let` or `const` instead of `var`

**Comment by indremak** [comment]
@vaibhavviking thank you for PR 👍  I think that this song is copyrighted. You should use sounds that are not copyrighted. You can find more information here: https://gamedev.stackexchange.com/questions/14/where-can-i-find-free-music-for-my-game. 
When you update a PR, provide a link to the music you used, so I can check if it is not copyrighted.

File: index.js (hunk 1 above)
Comment (vaibhavviking) [reply index.js]: https://incompetech.filmmusic.io/song/3358-amazing-plan

Comment (vaibhavviking) [reply index.js]: i have changed var to let . ignore the typo in the :P

**Comment by vaibhavviking** [comment]
any others changes needed?

**Pull request merged**